base/accumulate.jl

```diff
@@ -243,49 +228,45 @@ cumprod(itr) = accumulate(mul_prod, itr)
 Cumulative operation `op` along the dimension `dims` of `A` (providing `dims` is optional
 for vectors). An initial value `init` may optionally be provided by a keyword argument. See
 also [`accumulate!`](@ref) to use a preallocated output array, both for performance and
-to control the precision of the output (e.g. to avoid overflow). For common operations
-there are specialized variants of `accumulate`, see: [`cumsum`](@ref), [`cumprod`](@ref)
+to control the precision of the output (e.g. to avoid overflow).
+
+For common operations there are specialized variants of `accumulate`,
+see [`cumsum`](@ref), [`cumprod`](@ref). For a lazy version, see
+[`Iterators.accumulate`](@ref).
 
 !!! compat "Julia 1.5"
     `accumulate` on a non-array iterator requires at least Julia 1.5.
 
 # Examples
 ```jldoctest
-julia> accumulate(+, [1,2,3])
-3-element Vector{Int64}:
+julia> accumulate(+, Int8[1,2,3])
+3-element Vector{Int8}:
```

Comment:
Perhaps revert that:
```suggestion
julia> accumulate(+, [1,2,3])
3-element Vector{Int64}:
```